go/enclave/events/subscription_manager.go

```diff
@@ -151,7 +151,6 @@ func (s *SubscriptionManager) GetSubscribedLogsForBatch(batch *core.Batch, recei
 			if relevant {
 				relevantLogsForSub = append(relevantLogsForSub, logItem)
 			}
-			s.logger.Info(fmt.Sprintf("Subscription %s. Account %s. Log %v. Extracted addresses: %v. Relevant: %t", id, sub.Account, logItem, userAddrs, relevant))
```

Comment:
why is this gone?
This log is super useful.

Reply:
Good point, putting it back in.
Seemed like the log was spinning before fixing the p2p thing.
<img width="1427" alt="image" src="https://github.com/obscuronet/go-obscuro/assets/4471462/df41409c-85fa-4346-843e-6fee4c30a666">